include/eve/detail/function/simd/common/interleave.hpp

```diff
@@ -16,16 +16,16 @@ namespace eve::detail
   // N-ary interleave
   // This is the main one as it is easier to specify optimisation on this interface than on tuple
   //================================================================================================
-  template<simd_value T, std::same_as<T>... Ts>
-  EVE_FORCEINLINE auto interleave_(EVE_SUPPORTS(cpu_),T v0, Ts... vs) noexcept
+  template<callable_options OO, simd_value T, std::same_as<T>... Ts>
+  EVE_FORCEINLINE auto interleave_(EVE_REQUIRES(cpu_), OO const&, T v0, Ts... vs) noexcept
```

Comment:
Are we passing callable options by const& Is there any reason to? 

I'd expect by value but maybe it's important?

Reply:
I think that this is optimized out anyway here, as only the type is used

Reply:
I think it is irrelevant as we carry mostly type inforamtions. I guess I was sloppy and jtlap mimicked me.
We can later have a cleaning session to be homogeneous